sdk/identity/azure-identity/azure/identity/aio/_credentials/managed_identity.py

```diff
@@ -46,27 +49,24 @@ def __init__(self, client_id: "Optional[str]" = None, **kwargs: "Any") -> None:
 
 
 class _AsyncManagedIdentityBase(_ManagedIdentityBase):
-    def __init__(self, endpoint: str, config: "Optional[Configuration]" = None, **kwargs: "Any") -> None:
-        super().__init__(endpoint=endpoint, config=config, client_cls=AsyncAuthnClient, **kwargs)
+    def __init__(self, endpoint: str, **kwargs: "Any") -> None:
+        super().__init__(endpoint=endpoint, client_cls=AsyncAuthnClient, **kwargs)
 
     @staticmethod
-    def _create_config(**kwargs: "Any") -> "Configuration":  # type: ignore
-        """Build a default configuration for the credential's HTTP pipeline.
-
-        :rtype: :class:`azure.core.configuration`
-        """
+    def _create_config(**kwargs: "Any") -> "Configuration":
+        """Build a default configuration for the credential's HTTP pipeline."""
         return _ManagedIdentityBase._create_config(retry_policy=AsyncRetryPolicy, **kwargs)
 
 
 class ImdsCredential(_AsyncManagedIdentityBase):
     """Asynchronously authenticates with a managed identity via the IMDS endpoint.
 
-    :param config: optional configuration for the underlying HTTP pipeline
-    :type config: :class:`azure.core.configuration`
+    Keyword arguments:
+        - **client_id** (str): ID of a user-assigned identity. Leave unspecified to use a system-assigned identity.
```

Comment:
Are there any breadcrumbs to what the system-assigned identity is?  Does it ever matter?  (I'm thinking in terms of if one needs to know what this would present as for some service side use)